ui/components/package.json

```diff
@@ -23,6 +23,7 @@
     "lint:fix": "eslint --fix src --ext .ts,.tsx"
   },
   "dependencies": {
+    "@perses-dev/core": "^0.4.0",
```

Comment:
are you sure you want to remove the dependency in the `component` package too ?

Reply:
This is adding it to components package.json right? Intention was to make core a dependency of components since LineChart [imports useDeepMemo here](https://github.com/perses/perses/blob/v0.4.0/ui/components/src/LineChart.tsx#L15).

Reply:
ah sorry I read it wrongly. I thought it was removed :(

Reply:
no worries!